pages/attacks/Password_Spraying_Attack.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: col-sidebar
+title: Password Spraying Attack
+author: Rishu Ranjan
+contributors: 
+permalink: /attacks/Password_Spraying_Attack
+tags: attack, password spraying
+---
+
+{% include writers.html %}
+
+## Description
+**Password spraying** is a type of brute force attack. In this attack, an attacker will brute force on list of usernames with default password on the application. 
+For example, an attacker will user one password(say, Secure@123) against many different accounts on a application to avoid account lockouts that would normally occur when brute forcing a single account with many passwords.
```

Comment:
```suggestion
For example, an attacker will use one password (say, Secure@123) against many different accounts on a application to avoid account lockouts that would normally occur when brute forcing a single account with many passwords.
```